### ISSUE TITLE
Accessibility: MAS40B: Inspect: 'Name' property is empty for  "Search results" pane on Extensions sidebar area.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17113]
- VSCode Version: 1.6.1
- OS Version:1607

**Steps to Reproduce:**
      1.Open Visual studio code editor.
      2.Using "Ctrl+Shift+X",key navigate to 'Extensions' window.
      3. Tab to "Search results" pane present below 'Search Extensions in Marketplace' search edit box.
      4. Using Inspect, observe its "Name" property. 
**Expected Result:**
      "Name" property should not be empty. Pane should have a meaningful and appropriate name.
**Actual Result:**
      'Name' property is empty for the Search Results pane.
      Narrator Info:- Narrator announces as "Tab selected, requires selection contains 9 items, Starting".
      NVDA Info:- NVDA focus goes to the pane but announces nothing.
**User Impact**
Users relying on screen readers will get confused if the name property is not meaningful and inappropriate.

![search results pane name empty](https://cloud.githubusercontent.com/assets/22956130/21139063/21ead11c-c157-11e6-9e1f-80f237c63f81.png)

**Comment by WiproPSAccTest** [comment]
@joaomoreno  Here the issue related to side bar area Inspect properties. Where the "Name" value is missing. Bug#17242 related to options present in the extension side bar area. In bug 17242 Inspect properties are available but NVDA was not reciting the name.
